core/plugins/org.fusesource.ide.foundation.ui/src/org/fusesource/ide/foundation/ui/views/ColumnViewSupport.java

```diff
@@ -301,6 +303,7 @@ protected void addToolBarActions(IToolBarManager manager) {
 
 	protected void removeToolBarActions(IToolBarManager manager) {
 		Menus.removeAction(manager, toolBarActions);
+		Menus.removeAction(manager, refreshAction);
```

Comment:
Because I forgot to clean the code correctly after a lot of tested paths :s 
Good catch